Sorting Algorithms/Merge Sort Algorithm/Python/MergeSort.py

```diff
@@ -44,7 +45,7 @@ def merge(left, right):
 		return left
 	
 	# Compare first element and merge the remaining elements of the part (with smaller element) with other part
-	if left[0] < right[0]
+	if left[0] < right[0]:
 		return left[0] + merge(left[1:], right)
 		
 	return right[0] + merge(left, right[1:])
```

Comment:
You are returning an ```int``` here and a ```list``` in line 45, so this line gives an error saying ``` TypeError: unsupported operand type(s) for +: 'int' and 'list' ``` ... And please implement the merge logic using loops as you did earlier. This seems fine but calls ```merge``` too many times for large input.